src/sentry/models/group.py

```diff
@@ -241,6 +241,15 @@ class Meta:
 
     __repr__ = sane_repr('project_id')
 
+    def __getstate__(self):
+        state = Model.__getstate__(self)
+        state['message'] = state.pop('search_message', None)
+        return state
+
+    def __setstate__(self, state):
+        state['search_message'] = state.pop('message')
+        Model.__setstate__(self, state)
```

Comment:
Do these (and the ones on Event) only exist for backwards compatibility and pickling for existing workers during deploy? If so I think we should note their purpose because hopefully one day we can migrate away from doing this?

Reply:
Actually, won't we *have* to change that when we want to rename `real_message`?

Reply:
@bretthoerner it's mostly for during the deploy but also because of potential reverts and more annoying whatever people encounter on prem. As far as I can tell we're locked into this until we stop pickling the world.

And yes, we likely need to find a different key for `real_message` then too.